examples/Makefile

```diff
@@ -35,34 +49,101 @@ opencv:
 	pachctl create pipeline -f opencv/montage.json
 	pachctl put file images@master -i opencv/images.txt
 	pachctl put file images@master -i opencv/images2.txt
-#	pachctl put file images@master -i opencv/images3.txt
+
+opencv-testing: 
+	pachctl create repo images
```

Comment:
should we add the following  before the `pachctl create repo images`
```
pachctl delete repo images
pachctl delete pipeline edges
pachctl delete pipeline montage
```
If the pipeline/repo exist this target will drop an error.

Reply:
I added targets to delete each one.  I think having it fail is fine, because an automatic *destructive* action isn't desirable. You can't undelete a repo or a pipeline, and you might lose valuable test data if you accidentally hit "make opencv" when your config is pointing at the wrong cluster.